docs/pages/reference/cli.mdx

```diff
@@ -2013,3 +2013,221 @@ $ tsh ls --search=staging,mac
+   --token=00000000000000000000000000000000 \
+   --join-method=token \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=auth.example.com:3025
+```
+
+</ScopedBlock>
+
+### tbot init
+
+Allows certificates to be written to disks other than a Machine ID client, configuring either file or POSIX ACL permissions.
+This allows you to lock down access to Machine ID's short-lived certificates from other users or applications on the system.
+
+#### Flags
+
+| Flag                | Description                                                                                                        |
+|---------------------|--------------------------------------------------------------------------------------------------------------------|
+| `-d/--debug`        | Enable verbose logging to stderr.                                                                                  |
+| `-c/--config`       | Path to a configuration file.                                                                                      |
+| `--destination-dir` | Directory to write short-lived machine certificates to.                                                            |
+| `--owner`           | Defines the Linux `user:group` owner of `--destination-dir`. Defaults to the Linux user running `tbot` if unspecified. |
+| `--bot-user`        | Enables POSIX ACLs and defines the Linux user that can read/write short-lived certificates to `--destination-dir`. |
+| `--reader-user`     | Enables POSIX ACLs and defines the Linux user that will read short-lived certificates from `--destination-dir`.    |
+| `--init-dir`        | If using a config file and multiple destinations are configured, controls which destination dir to configure.      |
+| `--clean`           | If set, remove unexpected files and directories from the destination.                                              |
+
+#### Examples
+
+**Example using file permissions.**
+
+The following command highlights how to set permissions with tbot through Linux groups, using the user and group `jenkins:jenkins`.
+If running `tbot` as the Linux user `root`, use the following invocation of
+`tbot init` to initialize the short-lived certificate directory
+`/opt/machine-id` with owner `jenkins:jenkins`.
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --owner=jenkins:jenkins
+```
+
+**Example using POSIX ACLs.**
+
+If running `tbot` as the Linux user `teleport`, use the following invocation of
+`tbot init` to initialize the short-lived certificate directory
+`/opt/machine-id` with owner `teleport:teleport` but allow `jenkins` to read
+from `/opt/machine-id`.
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --bot-user=teleport \
+    --reader-user=jenkins
+```
+
+
+### tbot db
+
+Allows connections to databases using native clients and queries database information. This is best used for testing and validation purposes;
```

Comment:
```suggestion
Connects to databases using native clients and queries database information. This is best used for testing and validation purposes;
```

nit: I would rephrase like this, since `tbot db` isn't allowing connections, it's just connecting